python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/port-channel-interfaces.j2

```diff
@@ -258,13 +258,8 @@ interface {{ port_channel_interface.name }}
 {%     if port_channel_interface.l2_protocol.forwarding_profile is arista.avd.defined %}
    l2-protocol forwarding profile {{ port_channel_interface.l2_protocol.forwarding_profile }}
 {%     endif %}
-{%     if port_channel_interface.flow_tracker.hardware is arista.avd.defined %}
```

Comment:
Not in correct order
{%     if port_channel_interface.logging.event.link_status is arista.avd.defined(true) %}
   logging event link-status
{%     elif port_channel_interface.logging.event.link_status is arista.avd.defined(false) %}
   no logging event link-status
{%     endif %}
{%     if port_channel_interface.logging.event.storm_control_discards is arista.avd.defined(true) %}
   logging event storm-control discards
{%     elif port_channel_interface.logging.event.storm_control_discards is arista.avd.defined(false) %}
   no logging event storm-control discards
{%     endif %}

Reply:
Below once also not in correct order

{%     if port_channel_interface.type is arista.avd.defined("routed") %}
   no switchport

Reply:
please revisit all switchport commands as there are many config out of order

Reply:
wan2-dc1(config-if-Po1)#sh ac
interface Port-Channel1
   description asd
   shutdown
   mtu 2100
   logging event link-status
   bgp session tracker mahesh
   switchport access vlan 101
   switchport trunk native vlan tag
   switchport phone vlan 101
   switchport phone trunk tagged
   switchport trunk allowed vlan 101
   switchport mode trunk
   switchport trunk group aaa
   switchport trunk group bbb
   switchport trunk group ccc
   no switchport
   encapsulation dot1q vlan 101
   switchport trunk private-vlan secondary
   switchport pvlan mapping 101
   logging event storm-control discards

Reply:
Fixed all!